### ISSUE TITLE
Too long file names are not readable in "My Files"

### ISSUE THREAD

**Issue by Eldrile** [InseeFrLab/onyxia#672]
In  "My Files", the file name displays only 21 characters and there is no option to see it in full, only to copy it.

**Comment by garronej** [comment]
@Eldrile,  

Agreed, this is not great. There is a ton of stuff that needs reworking in the exporer. We'll make a big update in the comming month.

**Comment by odysseu** [comment]
Hi, when doing this repairing, it might be the time to go around https://github.com/InseeFrLab/onyxia/issues/529 and https://github.com/InseeFrLab/onyxia/issues/519 as well :)
